LICENSE.txt

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2019 Roman Dubrovsky
+Copyright (c) 2019 datarockets
```

Comment:
@dzhlobo is it ok? Ir should we set something else?

Reply:
Looks like the same was for dotfiles https://github.com/datarockets/dotfiles/blob/master/LICENSE#L3